src/components/checkboxes/_checkboxes.scss

```diff
@@ -6,6 +6,7 @@
 
     display: block;
     position: relative;
+    min-height: 40px;
```

Comment:
Note that this does change the spacing between checkboxes, from:

<img width="984" alt="screen shot 2018-02-09 at 14 38 24gmt" src="https://user-images.githubusercontent.com/121939/36032807-10e4a490-0da7-11e8-83a0-e80a9d9997c6.png">


to: 

<img width="980" alt="screen shot 2018-02-09 at 14 38 34gmt" src="https://user-images.githubusercontent.com/121939/36032816-179fd5fc-0da7-11e8-9005-677a512c279a.png">

I believe that the new behaviour is actually the 'correct' behaviour, but we should check with @dashouse when he's back.

Reply:
Ran this past @dashouse and he's okay with it 👍